LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright (c) 2015, Mat Groves
```

Comment:
Careful, the spine's runtime code (though adapted) is in here. Make sure the Spine license and associated copyrights are also present. LIkely best to put into a `SPINE-LICENSE` file.